rules/src/main/scala/fix/SortWith.scala

```diff
@@ -0,0 +1,30 @@
+package fix
+
+import scala.meta.Import
+
+sealed trait ImportOrdering extends Ordering[Import] {
+
+  protected def strFirstImport(imp: Import): String =
+    imp.children.head.syntax
+}
+
+class DefaultSort extends ImportOrdering {
+
+  override def compare(x: Import, y: Import): Int =
+    strFirstImport(x).compareTo(strFirstImport(y))
+}
+
+class WildcardAndGroupFirstSort extends ImportOrdering {
+
+  private def transformForSorting(imp: Import): (String, String) = {
+    val strImp = strFirstImport(imp)
+    (strImp, strImp.replaceAll("_", "\0").replaceAll("\\{.+\\}", "\1"))
```

Comment:
String#replaceAll compiles the first parameter into a `java.util.regex.Pattern`. This can have an impact on large code bases. You probably want to compile the Patterns once and use `Pattern.matcher(string).replaceAll(replacement)` (which `String#replaceAll` boils down to).